README.rst

```diff
@@ -136,6 +144,14 @@ Run quality assurance tests with::
 
     ./run-qa-checks
 
+You can run all unit tests by launching the dedicated script::
+
+    ./runtests
```

Comment:
I see quite some output like:

```
TestNetJSON.test_wifi_interfaces_stats_include ... {"interfaces":[{"mac":"00:00:00:00:00:00","type":"wireless","up":true,"name":"mesh1","wireless":{"noise":-87,"ssid":"meshID","country":"00","clients":[{"mac":"00:00:00:00:00:00","noise":0,"ht":true,"vht":false,"signal":-67,"wmm":true,"authorized":true,"mfp":true,"auth":true}],"channel":40,"tx_power":20,"mode":"802.11s","signal":-76,"frequency":5200},"multicast":true,"statistics":{"rx_frame_errors":0,"rx_crc_errors":0,"tx_heartbeat_errors":0,"rx_over_errors":0,"rx_errors":0,"tx_packets":138182411,"tx_carrier_errors":0,"rx_packets":94311062,"rx_length_errors":0,"tx_errors":0,"tx_aborted_errors":0,"tx_window_errors":0,"tx_bytes":151599685066,"collisions":0,"rx_missed_errors":0,"rx_fifo_errors":0,"rx_dropped":0,"tx_fifo_errors":0,"rx_compressed":0,"multicast":0,"tx_compressed":0,"rx_bytes":24917085991,"tx_dropped":0},"mtu":1500},{"mac":"00:00:00:00:00:00","type":"ethernet","up":true,"txqueuelen":1000,"link_supported":["10baseT-H","10baseT-F","100baseT-H","100baseT-F","1000baseT-F"],"name":"wan","speed":"1000F","multicast":true,"mtu":1500},{"mac":"00:00:00:00:00:00","type":"wireless","up":true,"txqueuelen":1000,"name":"wlan1","wireless":{"country":"00","noise":0,"ssid":"ssid","channel":13,"tx_power":20,"mode":"access_point","signal":-33,"frequency":5180},"multicast":true,"statistics":{"tx_crc_errors":0,"tx_heartbeat_errors":0,"tx_over_errors":0,"rx_errors":0,"tx_packets":0,"tx_carrier_errors":0,"rx_packets":2367515,"tx_length_errors":0,"tx_missed_errors":0,"tx_frame_errors":0,"tx_errors":0,"tx_aborted_errors":0,"tx_window_errors":0,"tx_bytes":0,"collisions":0,"rx_fifo_errors":0,"rx_dropped":0,"tx_fifo_errors":0,"rx_compressed":0,"multicast":0,"tx_compressed":0,"rx_bytes":531596854,"tx_dropped":0},"mtu":1500},{"mac":"00:00:00:00:00:00","type":"wireless","up":true,"txqueuelen":1000,"name":"mesh0","wireless":{"noise":0,"ssid":"meshID","country":"00","clients":[{"mac":"00:00:00:00:00:00","noise":-87,"ht":false,"vht":true,"signal":-76,"wmm":true,"authorized":true,"mfp":true,"auth":true}],"channel":11,"tx_power":20,"mode":"802.11s","signal":-67,"frequency":2462},"multicast":true,"mtu":1500},{"mac":"00:00:00:00:00:00","type":"wireless","up":true,"txqueuelen":1000,"name":"wlan0","wireless":{"country":"00","noise":0,"ssid":"OpenWRT","channel":13,"tx_power":20,"mode":"station","signal":-62,"frequency":2472},"multicast":true,"statistics":{"rx_frame_errors":0,"rx_crc_errors":0,"tx_heartbeat_errors":0,"rx_over_errors":0,"rx_errors":0,"tx_packets":2367747,"tx_carrier_errors":0,"rx_packets":198,"rx_length_errors":0,"tx_errors":0,"tx_aborted_errors":0,"tx_window_errors":0,"tx_bytes":531641723,"collisions":0,"rx_missed_errors":0,"rx_fifo_errors":0,"rx_dropped":0,"tx_fifo_errors":0,"rx_compressed":0,"multicast":0,"tx_compressed":0,"rx_bytes":25967,"tx_dropped":0},"mtu":1500}],"type":"DeviceMonitoring","general":{"hostname":"08-00-27-56-92-F5"},"resources":{"disk":{},"load":[0.37,0.95,1.23]}}
``` 

Can this extra output be avoided? If you remember, we did some work in the python module to suppress noisy output because some new contributors gets confused and think that output is because of errors.